src/helpers/pollUpdates.ts

```diff
@@ -2,36 +2,40 @@ import { updateAvailable } from '../app/notifications/actions';
 import { Store } from '../app/types';
 import { APP_ENV, RELEASE_ID } from '../config';
 
+export type Cancel = () => void;
+
+export const poll = (store: Store, cancel: Cancel) => async() => {
+  const release = await fetch('/rex/release.json')
+    .then((response) => response.json())
```

Comment:
What happens when a non-200 response is returned? Oh, `response.json()` will throw an error & be caught by `.catch(...)`. ok 😄